0028-web-monetization/0028-web-monetization.md

```diff
@@ -140,12 +136,12 @@ Dispatched once the first ILP packet with a non-zero amount has been fulfilled b
 {
   detail: {
     paymentPointer: String,
-    requestId: String
+    id: String
```

Comment:
What's the rationale for changing this field? This turns this into a breaking change